### PR TITLE
chore: bump library to support Aggregator path 0.2.0

### PR DESCRIPTION
**Pull request by yomarion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,5 @@ This subgraph indexes the price aggregators updates for easy query
 1. Create the subgraph on TheGraph hosted service
 2. clone this repo
 3. `yarn`
-4. `TOKEN=YOUR_TOKEN ./scripts/deploy.sh fantom`
\ No newline at end of file
+4. `yarn codegen subgraph.fantom.yaml`
```
Comment (yomarion) [inline README.md:9]: I'm quite sure this is mandatory but I'd appreciate a check. We can adapt the internal documentation accordingly.

File: subgraph.avalanche.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+specVersion: 0.0.4
```
Comment (yomarion) [inline subgraph.avalanche.yaml:1]: I assume that we just all forgot to commit and push local updates of the lib + supported networks.

File: src/aggregator.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { Address, log, store } from "@graphprotocol/graph-ts";
-import { AggregatorUpdated } from "../generated/Aggregator/AggregatorContract";
+import { AggregatorUpdated } from "../generated/Aggregator-0_2_0/AggregatorContract_0_2_0";
```
Comment (benjlevesque) [inline src/aggregator.ts:2]: I don't understand this? isn't it the same entity for both contracts?

**Review by benjlevesque — APPROVED** [review]

File: src/aggregator.ts (hunk 3 above)
Comment (yomarion) [reply src/aggregator.ts:2]: It's the generated file paths

**Review by leoslr — APPROVED** [review]

**Pull request merged**